openstax-poland/adaptarr-front
# Allow clearing local indexeddb for specific module

close #209 

Add new button in helpdesk screen
![obraz](https://user-images.githubusercontent.com/31478212/69957017-fc48c700-1501-11ea-9ab2-c39eda12c863.png)

Select drafts which you would like to remove from IndexedDB
![obraz](https://user-images.githubusercontent.com/31478212/69957070-1682a500-1502-11ea-8288-84f2b3002777.png)

### Testing

1. Change something in draft (ex. insert text) and do not save it
2. Refresh window to confirm that change was saved in IndexedDB
3. Go to Helpdesk screen
4. Clear IndexedDB for this draft
5. Go to draft and confirm that change is no longer visible

This should work for both: document and glossary content ~~(those are displayed separately)~~
